### DISCUSSION TITLE
With Gotenberg 8.8 the resulting PDFs from libreoffice conversation are 3x larger

### DISCUSSION THREAD

**Discussion by phortx** [gotenberg/gotenberg#922] [Q&A]
Hi,

with the update to Gotenberg 8.8 I discovered that my PDFs that are generated via the libreoffice endpoint are 3x larger in file size despite the PDFs totally look the same as before. Do you have an idea why?

**Comment by gulien** [comment]
Hey 👋 ,

Might be related to the new form fields OR the form field `reduceImageResolution` that has been reset to its actual default value (e.g., false): https://github.com/gotenberg/gotenberg/releases/tag/v8.8.0

Does it scale down with `reduceImageResolution` set to `false`?

**Comment by phortx** [comment]
I will try it, thanks for the response :)

**Comment by phortx** [comment]
Yes changing `reduceImageResolution` to `true` reduces the size again. That's it. Thanks :)
